google-cloud-bigquery/acceptance/bigquery_helper.rb

```diff
@@ -65,16 +65,16 @@ def setup
       addl.include? :bigquery
     end
 
-    def self.run_one_method klass, method_name, reporter
-      result = nil
-      reporter.prerecord klass, method_name
-      (1..3).each do |try|
-        result = Minitest.run_one_method(klass, method_name)
-        break if (result.passed? || result.skipped?)
-        puts "Retrying #{klass}##{method_name} (#{try})"
-      end
-      reporter.record result
-    end
+    # def self.run_one_method klass, method_name, reporter
```

Comment:
Was this commented out for testing?

Reply:
Yes, thanks for catching this, I'll restore it.